lawnchair/src/ch/deletescape/lawnchair/override/CustomInfoProvider.kt

```diff
@@ -50,6 +50,16 @@ abstract class CustomInfoProvider<in T : ItemInfo>(val context: Context) {
         TODO("not implemented")
     }
 
+    open fun supportsBadgeVisible(info: T) = false
+
+    open fun setBadgeVisible(info: T, visible: Boolean) {
+        TODO("not implemented")
+    }
+
+    open fun getBadgeVisible(info: T): Boolean {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [UndocumentedPublicFunction - [getBadgeVisible] at /src/lawnchair/src/ch/deletescape/lawnchair/override/CustomInfoProvider.kt:59:5](https://app.codacy.com/app/divadsn/Lawnchair/pullRequest?prid=4718535)

Reply:
No docs here to be consistent with the rest of the class. I can add it to this method, or the entire class, if desired.